uint16[16] voltage
uint16 testmsg
uint16 voltage00
uint16 voltage01
uint16 voltage02
uint16 voltage03
uint16 voltage04
uint16 voltage05
uint16 voltage06
uint16 voltage07
uint16 voltage08
uint16 voltage09
uint16 voltage10
uint16 voltage11
uint16 voltage12
uint16 voltage13
